.github/workflows/publish-dockerhub.yml

```
@@ -0,0 +1,30 @@
+name: Publish docker image
+on:
+  release:
+    types:
+      - published
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push release tag
+        id: docker_build_tag
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: guiabolso/guiabolso-connector:${{ github.event.release.tag_name }}
```

Seria melhor eliminar o step ` Build and push latest release` e enviar as duas tags aqui.
```suggestion
          tags: |
             guiabolso/guiabolso-connector:${{ github.event.release.tag_name }}
             guiabolso/guiabolso-connector:latest
```